deltacat/compute/compactor/utils/io.py

```diff
@@ -207,3 +192,25 @@ def limit_input_deltas(
     logger.info(f"Input uniform delta count: {len(rebatched_da_list)}")
 
     return rebatched_da_list, hash_bucket_count, latest_stream_position
+
+
+def getLastCompactedDeltaStreamPosition(
```

Comment:
Use `snake_case` instead of `pascalCase`

```suggestion
def get_last_compacted_delta_stream_position(
```